### PR TITLE
cleaning repo

### PR DESCRIPTION
**Pull request by guenk**
repo clean

### CODE REVIEW

File: frontend/src/pages/Home.jsx [hunk 1]
```diff
@@ -1,36 +1,9 @@
-import Counter from "../components/Counter";
-import logo from "../assets/logo.svg";
 
 export default function Home() {
   return (
-    <header className="App-header">
-      <img src={logo} className="App-logo" alt="logo" />
-      <p>Hello Vite + React !</p>
```
Comment (Sax593) [inline frontend/src/pages/Home.jsx:8]: This is not professional

**Review by Sax593 — CHANGES_REQUESTED** [review]
The project will not use CSS, so all .css files must be deleted.

File: frontend/src/pages/Home.jsx [hunk 2]
```diff
@@ -1,36 +1,9 @@
...
 
-      <Counter />
+    <p>Hello ptite perruche !</p>
+
```
Comment (Sax593) [inline frontend/src/pages/Home.jsx]: This is not professional

**Review by Sax593 — DISMISSED** [review]
Good job

**Review by Sax593 — APPROVED** [review]
it's ok

File: frontend/src/pages/Home/Home.jsx [hunk 3]
```diff
@@ -0,0 +1,9 @@
+
+export default function Home() {
+  return (
+
+    <p></p>
+
+
+  );
```
Comment (LoicBrassart) [inline frontend/src/pages/Home/Home.jsx:8]: Prettier didn't run on this, you have a conf problem

**Review by LoicBrassart — APPROVED** [review]
Someunclean code (prtettier conf) but we'll fix it later

**Pull request merged**